lnrpc/websocket_proxy.go

```diff
@@ -264,10 +302,10 @@ func (p *WebsocketProxy) upgradeToWebSocketProxy(w http.ResponseWriter,
 					writeDeadline := time.Now().Add(
 						p.pongWait,
 					)
-					_ = conn.SetWriteDeadline(writeDeadline)
-
-					err := conn.WriteMessage(
-						websocket.PingMessage, nil,
+					err := conn.WriteControl(
+						websocket.PingMessage,
+						[]byte("are you there?"),
```

Comment:
The mesage here is jsut arbitrary?

Reply:
Yes. Extracted it into a constant and added a comment.